docs/content/posts/2025-09-20-release-1.20.3.md

```diff
@@ -0,0 +1,104 @@
+---
+title:  "Apache Flink 1.20.3 Release Announcement"
+date: "2025-09-20T00:00:00.000Z"
```

Comment:
Just for notice, we should adjust the dates everywhere to the actual release date.